### PR TITLE
Improvement to read the file header

### LINKED ISSUE #88: Improvement to read the file header
**Description:**
Currently the alternative to read just the header is by using a RegEx. This implementation enable the user to read the 1st line of the file and send it to further processing without processing the whole file.

### PR DESCRIPTION
**Pull request by Ajanthy**
FIxes : https://github.com/siddhi-io/siddhi-io-file/issues/88

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: component/src/main/java/io/siddhi/extension/io/file/FileSource.java [hunk 1]
```diff
@@ -231,9 +231,19 @@
...
                                 "header line or not. This can either have value true or false. If it's set to " +
-                                "`true` then it indicates a file contains a header line, and it will not process",
+                                "`true` then it indicates a file contains a header line, and it will not process.\n",
                         optional = true, defaultValue = "false",
-                        type = {DataType.BOOL}),
+                        type = {DataType.BOOL}
+                ),
+                @Parameter(
+                        name = "read.only.header",
+                        description = "This parameter used to specify a particular text file (eg: CSV) contains a " +
```
Comment (ramindu90) [inline component/src/main/java/io/siddhi/extension/io/file/FileSource.java]: @Ajanthy shall we rephrase this as "This parameter used to read only the header or the first line of a particular text file (eg: CSV). This is only applicable if the `mode` is `LINE`. If it's set to false, the full file content will be read line by line.

**Review by ramindu90 — CHANGES_REQUESTED** [review]

File: component/src/main/java/io/siddhi/extension/io/file/FileSource.java (hunk 1 above)
Comment (Ajanthy) [reply component/src/main/java/io/siddhi/extension/io/file/FileSource.java]: fixed in https://github.com/siddhi-io/siddhi-io-file/pull/91/commits/eda3cf79cdc3e1487776729e63271fb7f5061e77

**Review by ramindu90 — APPROVED** [review]
